File: Iss_blood_front/src/main/java/Model/FormularDonare.java
package Model;

public class FormularDonare {
    private String nume;
    private String prenume;
    private Sex sex;
    private String telefon;
    private String domiciliuLocalitate;
    private String domiciliuJudet;
    private String domiciliuAdresa;
    private String resedintaLocalitate;
    private String resedintaJudet;
    private String resedintaAdresa;
    String beneficiarFullName;
    String beneficiarCNP;
    GrupaSange grupa;
    RH rh;
    private short zileDisponibil; //de ex: 00011 = luni si marti

    private Status status;


    public FormularDonare(String nume, String prenume, Sex sex, String telefon,
                          String domiciliuLocalitate, String domiciliuJudet, String domiciliuAdresa,
                          String resedintaLocalitate, String resedintaJudet, String resedintaAdresa,
                          String beneficiarFullName, String beneficiarCNP, GrupaSange grupa, RH rh,
                          short zileDisponibil) {
        this.nume = nume;
        this.prenume = prenume;
        this.sex = sex;
        this.telefon = telefon;
        this.domiciliuLocalitate = domiciliuLocalitate;
        this.domiciliuJudet = domiciliuJudet;
        this.domiciliuAdresa = domiciliuAdresa;
        this.resedintaLocalitate = resedintaLocalitate;
        this.resedintaJudet = resedintaJudet;
        this.resedintaAdresa = resedintaAdresa;
        this.beneficiarFullName = beneficiarFullName;
        this.beneficiarCNP = beneficiarCNP;
        this.grupa = grupa;
        this.rh = rh;
        this.zileDisponibil = zileDisponibil;
        this.status = Status.IN_ASTEPTARE;
    }

    public String getBeneficiarFullName() {
        return beneficiarFullName;
    }

    public String getBeneficiarCNP() {
        return beneficiarCNP;
    }

    public GrupaSange getGrupa() {
        return grupa;
    }

    public RH getRh() {
        return rh;
    }

    public String getNume() {
        return nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public Sex getSex() {
        return sex;
    }

    public String getTelefon() {
        return telefon;
    }

    public String getDomiciliuLocalitate() {
        return domiciliuLocalitate;
    }

    public String getDomiciliuJudet() {
        return domiciliuJudet;
    }

    public String getDomiciliuAdresa() {
        return domiciliuAdresa;
    }

    public String getResedintaLocalitate() {
        return resedintaLocalitate;
    }

    public String getResedintaJudet() {
        return resedintaJudet;
    }

    public String getResedintaAdresa() {
        return resedintaAdresa;
    }

    public short getZileDisponibil() {
        return zileDisponibil;
    }

    public Status getStatus() {
        return status;
    }
}

File: Iss_blood_back/Model/formular_donare.py

class FormularDonare:
    def __init__(self, nume, prenume, sex, telefon, domiciliu_localitate, domiciliu_judet, domiciliu_adresa,
                 resedinta_localitate, resedinta_judet, resedinta_adresa, beneficiar_full_name,
                 beneficiar_CNP, grupa, rh, zile_disponibil, status="IN_ASTEPTARE", id=-1):
        self.zile_disponibil = zile_disponibil
        self.rh = rh
        self.grupa = grupa
        self.beneficiar_CNP = beneficiar_CNP
        self.beneficiar_full_name = beneficiar_full_name
        self.resedinta_adresa = resedinta_adresa
        self.resedinta_judet = resedinta_judet
        self.resedinta_localitate = resedinta_localitate
        self.domiciliu_adresa = domiciliu_adresa
        self.domiciliu_judet = domiciliu_judet
        self.domiciliu_localitate = domiciliu_localitate
        self.telefon = telefon
        self.sex = sex
        self.prenume = prenume
        self.nume = nume
        self.status = status
        self.id = id
